content/docs/codebase-overview.md

```diff
@@ -218,7 +218,7 @@ module.exports = ReactHostComponent;
 
 يطبق React نظام أحداث اصطناعي محايد لتصيير ويعمل مع كل من React DOM و React Native. 
 
-شفرة المصدر الخاصة به موجودة في [`packages/events`](https://github.com/facebook/react/tree/master/packages/react-events).
+يطبّق React نظام أحداث وهمي منفصل عن المصيّرين (rendrers) ويعمل مع كل من React DOM و React Native. شيفرته المصدرية موجودة في [`packages/react-events`](https://github.com/facebook/react/tree/master/packages/react-events).
```

Comment:
URL `page not found `: https://github.com/facebook/react/tree/master/packages/react-events

Reply:
It's like that in the main repo, I've filed an issue to fix it #reactjs/reactjs.org/issues/2493

Reply:
is not `legacy-events`

Reply:
idk, for now, I suggest keeping it as it is and we will fix it when merging commits from the bot next time